imagersite/photo/templates/base.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <link rel="stylesheet" href="/static/style.css">
+    {% block scripts %}
+    {% endblock %}
+    <title>DjangoImager</title>
+</head>
+<body class="text-center">
+<h1> DjangoImager </h1>
+{% block main %}
+{% endblock %}
+
+<nav>
+    <ul>
+        <li><a href="{% url 'profile_list' %}">Profiles</a></li>
+        {% if user.is_authenticated %}
+        <li><a href="{% url 'add_page' model='photos' %}">Add Photo</a></li>
+        <li><a href="{% url 'add_page' model='album' %}">Add Album</a></li>
+        <li><a href="{% url 'profile_page' user_id=user.id %}">My Profile</a></li>
+        {% endif %}
+        <li><a href="{% url 'library_page' %}">Library</a></li>
+        {% block nav %}
+        {% endblock %}
+    </ul>
+    Welcome! {% if user.is_authenticated %}{{ user }}{% else %} SignUp {% endif %}!&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+    <a href="{% url 'homepage' %}">home</a>
+    {% if user.is_authenticated %}
+    <a href="{% url 'auth_logout' %}?next=/">logout</a>
+    {% else %}
+    <a href="{% url 'auth_login' %}">login</a>
+    <a href="{% url 'registration_register' %}">register</a>
+    {% endif %}
+
+</nav><br><br>
+{% block content %}
+{% endblock %}
+<footer class="text-center"> Make it a Django moment. </footer>
+</body>
+</html>
```

Comment:
Why a second version of the `base.html` template?  Does this get used?  Or does the other one?